JavaScript/Core/quirks-i/changing-a-functions-context-with-fn-call-object.md

```diff
@@ -11,6 +11,8 @@ type: normal
 
 category: must-know
 
+standard: js.evaluate-expressions
```

Comment:
I added a standard for this one - https://github.com/sagelabs/standards/blob/master/javascript/core/use-and-manipulate-the-execution-context-of-a-function-in-javascript.md